src/libwalletqt/PendingTransaction.cpp

```diff
@@ -50,6 +50,16 @@ quint64 PendingTransaction::txCount() const
     return m_pimpl->txCount();
 }
 
+QList<QVariant> PendingTransaction::subaddrIndices() const
+{
+    std::vector<std::set<uint32_t>> subaddrIndices = m_pimpl->subaddrIndices();
+    QList<QVariant> result;
+    for (const auto& x : subaddrIndices)
+        for (uint32_t i : x)
+            result.push_back(i);
+    return result;
+}
```

Comment:
This looks like you'll get duplicates when having several txes. Maybe removing any would be better.